test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
> Since that's 0.6-only, this will similarly have to be made 0.6-only. Right?

I don't think it is necessary to enforce this in packages that depend on `DataArrays`. As long as we don't use new features added to `DataArrays` it is up to `DataArrays` to figure out which version can be installed. Which problem would be avoided by bumping the version requirement for `DataArrays`?

> Older versions of the package should be upper bounded at Julia 0.6 as a precaution, since the current state of affairs is known not to work on 0.6 due to DataArrays issues. That's what we had talked about with Tony in triage.

I know that Tony wants this but I'm not convinced that it matters much in practice. Probably my view will change when things are more mature but right now I can see two scenarios. If the upper bounds are added at the same time as a new version of the package is made available for, say 0.6, then when running `Pkg.update()`, you'll receive the upper bounds in METADATA **and** the updated package. In that situation, not having the bounds would have given the same result. The other scenario is a situation where the package doesn't have a working version for 0.6. Then adding the upper bounds trades a broken package for a complaining package manager. Most people I've discussed this with would prefer the broken package because of `Pkg`.